app/script_variant.py

```diff
@@ -145,6 +145,53 @@ def romanize_word_by_unidecode(word):
     romanized_word = unidecode(word)
     return [romanized_word.strip()]
 
+def is_chinese_person_name(word):
+    """
+    This method checks if a unicode string is a chinese person name by figuring out
+    if it contains valid a chinese surname and a given name.
+    
+    Return:
+        Boolean
+    """
+    surname, lastname  = split_chinese_name(word)
+    return surname is not None and lastname is not None
+
+def split_chinese_name(word):
+    """
+    This method tries to split a chinese name into two parts: surname and lastname
```

Comment:
Use family_name and given_name, which are terminology used in Person Finder.

"lastname" usually means surname in English, so it's confusing.